python/src/aiconfig/editor/client/src/shared/types.ts

```diff
@@ -63,15 +63,17 @@ export function aiConfigToClientConfig(aiconfig: AIConfig): ClientAIConfig {
   };
 }
 
-export type LogEvent = 
+export type LogEvent =
   | "ADD_PROMPT"
-  | "SAVE_BUTTON_CLICKED" 
+  | "SAVE_BUTTON_CLICKED"
   | "RUN_PROMPT_START"
-  | "RUN_PROMPT_CANCELED" 
-  | "RUN_PROMPT_ERROR" 
+  | "RUN_PROMPT_CANCELED"
+  | "RUN_PROMPT_ERROR"
   | "RUN_PROMPT_SUCCESS";
 
 // TODO: schematize this
 export type LogEventData = JSONObject;
 
 export type AIConfigEditorMode = "local" | "gradio" | "vscode";
+
+export type ThemeMode = "light" | "dark";
```

Comment:
Shouldn't we also allow `"system"`?

Reply:
Ok nm, "system" is purely Gradio convention. Both RN and Mantine use dark, light only: https://mantine.dev/hooks/use-color-scheme/#definition

Reply:
Ya, 'system' for us is equivalent to not passing anything since we already grab the system theme in the ThemeProvider